woocommerce/woocommerce-gutenberg-products-block
# Test PR #6661

This PR adds a plain text file to the repo, for testing https://github.com/woocommerce/woocommerce-blocks/pull/6661.

## Steps to test

1. Spin up a test site using https://jurassic.ninja/.
2. Install the custom WP-CLI command woo-test-environment using the following command:
```sh
wp package install git@github.com:nielslange/woo-test-environment.git
```
3. Spin up a WooCommerce test environment with this PR using the following command:
```sh
wp woo-test-environment setup --blocks=https://wcblocks.wpcomstaging.com/wp-content/uploads/woocommerce-gutenberg-products-block-6692.zip
```
4. Look up the WooCommerce Blocks folder of the site.
5. Verify that the WooCommerce Blocks folder contains the file `test-pr-6661.txt`.
